packages/envapt/tests/.env.037-required-reads

```
@@ -0,0 +1,20 @@
+# Fixture for 037-required-reads.test.ts
+
+SET_NUMBER=42
+SET_VALUE=hello
+DATABASE_URL=postgres://localhost/db
+GITHUB_REPOSITORY=owner/repo
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_REPOSITORY key should go before the SET_NUMBER key

(UnorderedKey)